src/ingestion/iam_credential_report.py

```diff
@@ -123,31 +123,37 @@ def get_data_worker(account):
                 break
             except Exception as e:
                 time.sleep(2)
-        policy = iam_client.get_account_password_policy()['PasswordPolicy']
+        try:
+            policy = iam_client.get_account_password_policy()['PasswordPolicy']
+        except Exception as e:
+            policy = {}
         report = r['Content'].decode('utf-8').split('\n')
         data = [i.split(',') for i in report]
         reports = [dict(zip(data[0], i)) for i in data[1:]]
         policy['AccountId'] = account
         reports_json = [json.dumps({**report, "account_id": account}) for report in reports]
-        sf_client = get_snowflake_client()
+        return {'report': reports_json, 'policy': json.dumps(policy)}
 
+def get_data(accounts_list):
+    start = datetime.datetime.now()
+    results_list=list(filter(None,Pool(4).map(get_data_worker, accounts_list)))
```

Comment:
this looks like you're repeating the same thing as you have in the other file? can you add it to a common file and import from there?